aiortc/rtcdatachannel.py

```diff
@@ -17,17 +17,26 @@ class RTCDataChannel(EventEmitter):
     :param: parameters: An :class:`RTCDataChannelParameters`.
     """
 
-    def __init__(self, transport, parameters, id=None):
+    def __init__(self, transport, parameters, send_open=True):
         super().__init__()
         self.__bufferedAmount = 0
         self.__bufferedAmountLowThreshold = 0
-        self.__id = id
+        self.__id = parameters.id
+        self.__negotiated = parameters.negotiated
```

Comment:
This isn't useful, as we already store parameters. Let's stay consistent.

Reply:
What do you mean? To use parameters directly in code? Should I also remove self.__id?

Reply:
Hm I was tired when I wrote it, keep it as is